test/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -aeuo pipefail
+
+echo "Running setup.sh"
+echo "Waiting until configuration package is healthy/installed..."
+"${KUBECTL}" wait configuration.pkg configuration-aws-eks --for=condition=Healthy --timeout 5m
+"${KUBECTL}" wait configuration.pkg configuration-aws-eks --for=condition=Installed --timeout 5m
+
+echo "Waiting until network configuration package is healthy/installed..."
+"${KUBECTL}" wait configuration.pkg upbound-configuration-aws-network --for=condition=Healthy --timeout 5m
+"${KUBECTL}" wait configuration.pkg upbound-configuration-aws-network --for=condition=Installed --timeout 5m
```

Comment:
nit: we can switch to a universal `wait configuration.pkg --all` similar to https://github.com/upbound/configuration-aws-database/blob/main/test/setup.sh#L6